src/content-tags-drawer/ContentTagsDrawer.tsx

```diff
@@ -60,7 +61,13 @@ const TaxonomyList = ({ contentId }: TaxonomyListProps) => {
               size="inline"
               variant="link"
               className="text-info-500 p-0 enable-taxonomies-button"
-              onClick={() => navigate('/taxonomies')}
+              onClick={() => {
+                if (window.top && window.top !== window) {
+                  window.top.location.href = `${getConfig().COURSE_AUTHORING_MICROFRONTEND_URL}/taxonomies`;
+                } else {
+                  navigate('/taxonomies');
+                }
+              }}
```

Comment:
Thanks for this! We never really test without any taxonomies so hadn't encountered this.

The bug is probably a non-issue on master anymore, because everyone is expected to use the new unit page and the legacy page is now off by default. But it's likely an issue in Sumac.

I notice that this is currently using a `<Button` but that's incorrect for a link that triggers a navigation. This should really just be a `<Link>` component (from react-router-dom) and then I think you can simplify it to `<Link to="/taxonomies" target="_top" className="...">` and get rid of all the javascript.